content/applications/general/iot/config/updating_iot.rst

```diff
@@ -73,6 +73,70 @@ Lastly, click :guilabel:`Flash`, and wait for the process to finish.
    An alternative software for flashing the micro SD card is `Raspberry Pi Imager
    <https://www.raspberrypi.com/software/>`_.
 
+Windows IoT update
+==================
+
+Windows virtual :abbr:`IoT (Internet of Things)` box may occasionally need an update to work
+properly. The following processes are the un-installation and re-installation of the Windows virtual
+:abbr:`IoT (Internet of Things)` box.
+
+.. important::
+   Before un-installation of a previous version of the Windows virtual :abbr:`IoT (Internet of
+   Things)` box, check `Odoo's download page <https://odoo.com/download>`_ to ensure a newer version
+   of Odoo's :abbr:`IoT (Internet of Things)` box is available.
+
+Uninstalling Windows IoT
```

Comment:
I don't think this section should belong here. I think it would be better elsewhere and put a .seealso here

Reply:
@lse-odoo 

You mentioned on discord that this is a required step to updating the windows IoT. If this isn't true we can remove it, otherwise I think it's good to leave in.

Reply:
It is indeed required to remove then reinstall. My point is rather that in my opinion the uninstall process should be in a more appropriate page/section (troubleshooting maybe ?)